specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/stable/2022-11-08/examples/ClusterCreateSingleNode.json

```diff
@@ -0,0 +1,141 @@
+{
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "TestGroup",
+    "clusterName": "testcluster-singlenode",
+    "api-version": "2022-11-08",
+    "parameters": {
+      "location": "westus",
+      "properties": {
+        "administratorLoginPassword": "password",
+        "postgresqlVersion": "15",
+        "citusVersion": "11.3",
+        "enableHa": true,
+        "enableShardsOnCoordinator": true,
+        "coordinatorServerEdition": "GeneralPurpose",
+        "coordinatorStorageQuotaInMb": 131072,
+        "coordinatorVCores": 2,
```

Comment:
Should probably use coordinator vCore value more than 2 if we are giving an example for GeneralPurpose

Reply:
Updated to 8 vCores